frontend/src/patientApp/timeOfTest/ToS.tsx

```diff
@@ -23,12 +23,16 @@ function ToS() {
       </h2>
       <ul>
         <li>
-          <b>{t("testResult.tos.document.definitions.l0.title")}</b>{" "}
-          {t("testResult.tos.document.definitions.l0.definition")}
+          <>
```

Comment:
out of curiosity, is there a reason we need to wrap this in `<>` now?

Reply:
Typescript was giving me errors about passing more than one node to the upper component, so I just added a fragment to get rid of the error